### ISSUE TITLE
xsbt-web-plugin 1.0.0 is incompatible with scalatra-sbt 0.3.5

### ISSUE THREAD

**Issue by seratch** [scalatra/scalatra-sbt#9]
We should fix that in the next release. If no one fix this issue, I'll send a pull request later.

https://github.com/earldouglas/xsbt-web-plugin

```
Caused by: java.lang.ClassNotFoundException: com.earldouglas.xsbtwebplugin.PluginKeys$
	at java.net.URLClassLoader$1.run(URLClassLoader.java:372)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:361)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:360)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at org.scalatra.sbt.ScalatraPlugin$.<init>(ScalatraPlugin.scala:15)
	at org.scalatra.sbt.ScalatraPlugin$.<clinit>(ScalatraPlugin.scala)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:340)
```

**Comment by dozed** [comment]
Using `webappDest` during `dist:stage` to stage the standalone distribution also includes `WEB-INF/lib`. This leads to problems, since the libraries are then contained twice in the distribution (in `./webapp/WEB-INF/lib` and `./lib` directory). This requires a better solution.

**Comment by dozed** [comment]
I think excluding the `WEB-INF/lib` and `WEB-INF/classes` when building a distribution is a simple but effective solution. See: https://github.com/dozed/scalatra-sbt/commit/6845051835a49e9d2ecc7c951770ca288041be44

**Comment by seratch** [comment]
@dozed Thank you for your attention to this matter. It would be nice upgrading scalatra-sbt to use web plugin 1.0 when Scalatra 2.4.0 release (the date of release is not decided yet).

**Comment by dozed** [comment]
@seratch It needs some testing, but I think it is stable. If you need help let me know.

**Comment by seratch** [comment]
Fixed 0.4.0 will be out soon (I'll do the release).
